### PR TITLE
[Doc] Reorganize set catalog & desc catalogs

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/data_source/catalog/iceberg_catalog.md [hunk 1]
```diff
@@ -840,13 +857,26 @@ Similar to the internal catalog of StarRocks, if you have the [CREATE DATABASE](
 >
 > You can grant and revoke privileges by using [GRANT](../../sql-reference/sql-statements/account-management/GRANT.md) and [REVOKE](../../sql-reference/sql-statements/account-management/REVOKE.md).
 
+[Switch to an Iceberg catalog](#switch-to-an-iceberg-catalog-and-a-database-in-it), and then use the following statement to create a Hive database in that catalog:
```
Comment (evelynzhaojie) [inline docs/data_source/catalog/iceberg_catalog.md:860]: hive? same for other sentences

File: docs/data_source/catalog/iceberg_catalog.md [hunk 2]
```diff
@@ -857,6 +887,8 @@ Similar to the internal databases of StarRocks, if you have the [DROP](../../adm
 
 When you drop an Iceberg database, the database's file path on your HDFS cluster or cloud storage will not be dropped along with the database.
 
+[Switch to an Iceberg catalog](#switch-to-an-iceberg-catalog-and-a-database-in-it), and then use the following statement to drop a Hive database in that catalog:
```
Comment (evelynzhaojie) [inline docs/data_source/catalog/iceberg_catalog.md:890]: hive

File: docs/data_source/catalog/iceberg_catalog.md [hunk 3]
```diff
@@ -943,17 +977,11 @@ You can specify the table attributes in the `"key" = "value"` format in `propert
    PARTITION BY (id,dt);
    ```
 
-3. Create a partitioned table named `partition_tbl`. The table consists of four columns, `v1`, `v2`, `k1`, and `k2`, of which `k1` and `k2` are defined as partition columns. At the same time, specify that StarRocks synchronously queries an existing table named `employee` and inserts the query result into the new table `partition_tbl`, as shown below:
+3. Query an existing table named `partition_tbl_1`, and create a partitioned table named `partition_tbl_2` into which the query result on `partition_tbl_1` is inserted. For `partition_tbl_2`, `id` and `dt` are defined as partition columns, as shown below:
```
Comment (evelynzhaojie) [inline docs/data_source/catalog/iceberg_catalog.md:980]: Query an existing table named `partition_tbl_1`, and create a partitioned table `partition_tbl_2` based on the query result of `partition_tbl_1`.

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by hellolilyliuyi — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by evelynzhaojie — APPROVED** [review]

**Review by hellolilyliuyi — APPROVED** [review]

**Comment by amber-create** [comment]
@mergify backport branch-3.2 branch-3.1

**Pull request merged**